lib/attach-middleware.js

```diff
@@ -13,12 +13,17 @@ function logError(err, req, res, next) {
   next(err);
 }
 
-function fixFilePaths(coverageData, fileLookup) {
-  coverageData.path = fileLookup[coverageData.path];
+function fixFilePaths(coverageData, fileLookup, configPath) {
+  let found = getConfig(configPath).coveredAddons.find((item) =>  coverageData.path.startsWith(item));
```

Comment:
so I'm essentially saying is if this file is in our covered addons then prepend a `../` to it